README.md

```diff
@@ -10,15 +10,15 @@ Gridicon renders a single svg icon based on an `icon` prop. It takes a size prop
 
 There's a gallery with all the available icons in http://automattic.github.io/gridicons/.
 
-```
-npm install gridicons --save
+```sh
+$ npm install gridicons --save
 ```
 
 #### Usage
 
 You can either import the whole iconset and decide at run-time which icon to use:
 
-```
+```js
```

Comment:
Shall we use `jsx` instead?

Reply:
I didn't know that existed. TIL thanks! :)